Sources/Widgets/Representations/WidgetRepresentation/index.js

```diff
@@ -161,6 +161,38 @@ function vtkWidgetRepresentation(publicAPI, model) {
     }
   };
 
+  function applyCoincidentTopologyParametersToMapper(
+    mapper,
+    { factor, offset }
+  ) {
+    mapper.setResolveCoincidentTopologyToPolygonOffset();
+    mapper.setRelativeCoincidentTopologyPolygonOffsetParameters(factor, offset);
+    mapper.setRelativeCoincidentTopologyLineOffsetParameters(factor, offset);
+    mapper.setRelativeCoincidentTopologyPointOffsetParameters(factor, offset);
+    console.log(mapper);
+  }
+
+  publicAPI.initializeMapper = (mapper) => {
+    applyCoincidentTopologyParametersToMapper(
+      mapper,
+      model.coincidentTopologyPolygonOffsetParameters
+    );
+    model.initializedMappers.push(mapper);
+    return mapper;
+  };
+
+  publicAPI.setCoincidentTopologyPolygonOffsetParameters = (parameters) => {
```

Comment:
How about `setCoincidentTopologyParameters` (because it is for PolygonOffset, LineOffset and PointOffset parameters)?

Further on, parameters could then be { PolygonOffset: {factor, offset}, LineOffset: {factor, offset})...)